src/GafferOFX/ParamInstance.cpp

```
@@ -0,0 +1,476 @@
+	}
+
+	typename PlugType::Ptr plug = new PlugType( parameterName, direction, defaultValue );
+
+	plug->setFlags( Gaffer::Plug::Dynamic, true );
+	PlugAlgo::replacePlug( plugParent, plug );
+
+	return plug.get();
+}
+}
+
+IntegerInstance::IntegerInstance( GafferOFX::EffectImageInstance* effect, const std::string& name, OFX::Host::Param::Descriptor& descriptor ) : OFX::Host::Param::IntegerInstance( descriptor ), m_effect( effect ), m_descriptor( descriptor )
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_effect

(cppcoreguidelines-pro-type-member-init)